src/command_modules/azure-cli-keyvault/azure/cli/command_modules/keyvault/_help.py

```diff
@@ -6,3 +6,8 @@
 from azure.cli.core.help_files import helps #pylint: disable=unused-import
 
 #pylint: disable=line-too-long
+
+helps['keyvault'] = """
+    type: group
+    short-summary: Safeguard and maintain control of keys and other secrets
```

Comment:
Fixed, and good point on mgmt plane; but bit wordy for text to show up on `az -h` and a point in time prob.